apps/heft/src/utilities/RushUtilities.ts

```diff
@@ -38,6 +38,11 @@ export function getRushCommonTempFolder(): string {
   return path.join(rushJsonFolder, 'common', 'temp');
```

Comment:
If a project has `splitWorkspace=false` AND that project has a lockfile in its project folder, Rush should print a warning about this to avoid confusion.